test/e2e/common/node/pods.go

```diff
@@ -328,7 +328,7 @@ var _ = SIGDescribe("Pods", func() {
 		}
 
 		gomega.Expect(lastPod.DeletionTimestamp).ToNot(gomega.BeNil())
-		gomega.Expect(lastPod.Spec.TerminationGracePeriodSeconds).ToNot(gomega.BeZero())
```

Comment:
Why change this? `BeZero` seems more simple?

Reply:
I think `gomega.BeNil()` is expected.

Reply:
adjusted to Bezero